src/utils/scene-url-utils.js

```diff
@@ -0,0 +1,21 @@
+import { isHubsSceneUrl, proxiedUrlFor } from "../utils/media-url-utils";
+
+export async function isValidGLB(url) {
+  return fetch(url).then(r => {
+    const reader = r.body.getReader();
+    return reader.read().then(result => {
+      reader.cancel();
+      return String.fromCharCode.apply(null, result.value.slice(0, 4)) === "glTF";
```

Comment:
this is a little weird, but i guess i can't think of a better way to do it. is there any chance we may get less than 4 characters in the first `read()`?

Reply:
Seems really unlikely that a valid GLB will have a first chunk that is less than 4 bytes, but I'll change it to iterate through the stream, just in case.